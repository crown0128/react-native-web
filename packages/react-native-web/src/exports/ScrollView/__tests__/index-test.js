/* eslint-env jasmine, jest */

describe('components/ScrollView', () => {
  test.skip('todo', () => {});
});
