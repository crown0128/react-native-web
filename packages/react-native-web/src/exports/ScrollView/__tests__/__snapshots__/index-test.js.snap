// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/ScrollView prop "refreshControl" with 1`] = `
<div
  id="refresh-control"
>
  <div
    class="css-view-1dbjc4n r-WebkitOverflowScrolling-150rngu r-flexDirection-eqz5dr r-flexGrow-16y2uox r-flexShrink-1wbh5a2 r-overflowX-11yh6sk r-overflowY-1rnoaur r-transform-1sncvnh"
    style="background-color: rgb(255, 0, 0);"
  >
    <div
      class="css-view-1dbjc4n"
    />
  </div>
</div>
`;

exports[`components/ScrollView prop "refreshControl" without 1`] = `
<div
  class="css-view-1dbjc4n r-WebkitOverflowScrolling-150rngu r-flexDirection-eqz5dr r-flexGrow-16y2uox r-flexShrink-1wbh5a2 r-overflowX-11yh6sk r-overflowY-1rnoaur r-transform-1sncvnh"
  style="background-color: rgb(255, 0, 0);"
>
  <div
    class="css-view-1dbjc4n"
  />
</div>
`;
